ocamltest/main.ml

```diff
@@ -152,9 +152,10 @@ let test_file test_filename =
   let test_build_directory_prefix =
     get_test_build_directory_prefix test_directory in
   let clean_test_build_directory () =
-    ignore
-      (Sys.command
-         (Filename.quote_command "rm" ["-rf"; test_build_directory_prefix]))
+    try
+      Sys.rm_rf test_build_directory_prefix
+    with Sys.Break as e -> raise e
```

Comment:
Is this special handling of the `Break` exception due to the potentially lengthy nature of this operation?

Reply:
Well, in a command-line tool, you never want to lose `Sys.Break` in internal handlers, yes. Imagine this is being done an NFS link or something which is timing out, for example.